stripe-ui-core/src/main/java/com/stripe/android/uicore/address/AddressRepository.kt

```diff
@@ -22,7 +21,6 @@ import kotlin.coroutines.CoroutineContext
  * Note: this repository is mutable and stateful. The address information saved within the Element
  * list will carry over to other screens.
  */
-@Singleton
```

Comment:
I tested out the address repository, couldn't find any reason why this scoping was here.